projects/common/src/navigation/navigation.service.ts

```diff
@@ -272,8 +272,8 @@ export class NavigationService {
     return this.platformLocation.href;
   }
 
-  public getHostUrl(): string {
-    return new URL(this.getAbsoluteCurrentUrl()).origin;
+  public getShareableUrl(path: string[], queryParams?: Dictionary<string>): string {
```

Comment:
Could we make this interface match some of the other ones? They're a little inconsistent but either taking NavParams like our main nav method
```
navigate(paramsOrUrl: NavigationParams | string)
```
or at least taking the query params as a Params object like getUrlTree (not sure what the type of UrlSegment is - if it's an alias for string, even better - if it needs to be constructed, string[] is fine)
```
getUrlTree(urlSegments: UrlSegment[], queryParams?: Params)
```

Reply:
done.